domain/removal/service/application.go

```diff
@@ -135,16 +136,113 @@ func (s *Service) RemoveApplication(
 		}
 	}
 
+	for _, m := range cascaded.MachineUUIDs {
```

Comment:
Question? Is the order important here? Must machines be before storage? If so Is there any value adding a comment here for that ?

Reply:
Just making it look like the other places. Removal jobs can be made in any order.